tests/basic/pos/ReflExt02A.hs

```diff
@@ -0,0 +1,14 @@
+-- | test reflection away from another imported module (ReflExt02B)
+
+{-@ LIQUID "--ple"         @-}
+{-@ LIQUID "--reflection" @-}
+
+module ReflExt02A where
+
+import ReflExt02B
+
+{-@ reflect myAdd @-}
+
+-- 3 + 2 + 1 = 6
+{-@ lemma :: {myAdd 3 2 = 6 } @-}
```

Comment:
It would be interesting to test reflection when there is a function named the same as the imported one.

Reply:
Done in a2bd4dd698e671a1b041909d20bc133cd73089c6